features/support/poor_mans_webmock.rb

```diff
@@ -4,7 +4,7 @@
 
 module Workers
   class PublishToHub < Base
-    def perform(sender_public_url)
+    def perform(sender_atom_url)
```

Comment:
Unused method argument - `sender_atom_url`. If it's necessary, use `_` or `_sender_atom_url` as an argument name to indicate that it won't be used. You can also write as `perform(*)` if you want the method to accept any arguments but don't care about them.